### PR TITLE
Update SSO Docs

### PR DESCRIPTION
**Pull request by benarent**
Updated as part of 4.3 testing.

### CODE REVIEW

File: examples/resources/oidc-connector.yaml [hunk 1]
```diff
@@ -1,52 +1,20 @@
 #
 # Example resource for an OIDC connector
-# We recommend using OIDC for G Suite, Auth0 and Keycloak
+# We recommend using oidc for G Suite, Auth 0 and Keyclock
```
Comment (webvictim) [inline examples/resources/oidc-connector.yaml:3]: This line needs to be reverted to what it was before
`oidc` -> `OIDC`
`Auth 0` -> `Auth0`
`Keyclock` -> `Keycloak`

File: docs/4.3/enterprise/ssh_sso.md [hunk 2]
```diff
@@ -192,8 +192,7 @@ SAML and OIDC types:
 | Microsoft |`display: Microsoft`|![microsoft](../img/teleport-sso/microsoft.png)|
 | Google |`display: Google`|![google](../img/teleport-sso/google.png)|
 | BitBucket | `display: Bitbucket` | ![bitbucket](../img/teleport-sso/bitbucket.png)|
-| OpenID | `display: OpenID` | ![bitbucket](../img/teleport-sso/openId.png)|
-| Custom Provider | `display: Homegrown SSO` | ![bitbucket](../img/teleport-sso/custom-content.png)|
+| OpenID | `display: okta` | ![Okta](../img/teleport-sso/openId.png)|
```
Comment (webvictim) [inline docs/4.3/enterprise/ssh_sso.md:195]: This should really be `display: Okta`

**Review by webvictim — COMMENTED** [review]
There's a lot of red in this diff - did we really want to delete the entire OIDC page?

File: examples/resources/oidc-connector.yaml (hunk 1 above)
Comment (benarent) [reply examples/resources/oidc-connector.yaml:3]: This was pull from the UI, so I'll also track this issue in another ticket.

**Comment by benarent** [comment]
> There's a lot of red in this diff - did we really want to delete the entire OIDC page?

We had duplicate content, I've moved SSO under the enterprise section.

File: examples/resources/oidc-connector.yaml [hunk 3]
```diff
@@ -1,52 +1,20 @@
 #
 # Example resource for an OIDC connector
-# We recommend using OIDC for G Suite, Auth0 and Keycloak
+# We recommend using OIDC for G Suite, Auth0 and KeyClock
```
Comment (webvictim) [inline examples/resources/oidc-connector.yaml:3]: `KeyClock` -> `Keycloak`

**Review by webvictim — APPROVED** [review]

**Pull request merged**